Dockerfile

```diff
@@ -1,46 +1,50 @@
-FROM alpine:3.7
+FROM alpine:3.6
```

Comment:
Why the downgrade?

Reply:
Faced some issue with bash apk's

Reply:
hrmm shouldn't need to downgrade. Can you share the error you had? I bet I've seen it before.

Reply:
```ERROR: ncurses-terminfo-6.0_p20171125-r0: Protocol error
(6/77) Installing readline (7.0.003-r0)
(7/77) Installing bash (4.4.19-r1)
ERROR: bash-4.4.19-r1: Protocol error
(8/77) Installing bash-completion (2.7-r3)
...
(14/77) Installing libcurl (7.59.0-r0)
ERROR: libcurl-7.59.0-r0: Protocol error
(15/77) Installing curl (7.59.0-r0)
...
(17/77) Installing drill (1.6.17-r6)
ERROR: ldns-1.6.17-r6: Protocol error
```